pipeline/relation_embedder/batcher/README.md

```diff
@@ -0,0 +1,66 @@
+# Batcher
+
+## Running Locally
+
+First, build it (from the base of the repo)
+`sbt "project batcher" ";stage"`
+
+### As a JAR
+
+You can pipe a bunch of paths to CLIMain, thus:
+
+`cat src/test/resources/paths.txt | java weco.pipeline.batcher.CLIMain`
+
+### As a Lambda
+
+You can run the Lambda version locally thus:
+
+Build the appropriate Docker
+
+`docker build --target lambda_rie -t lambda_batcher .`
+
+Run it with the port available
+
+`docker run -p 9000:8080 lambda_batcher`
+
+You can now post JSON SQS messages to it, e.g.
+
+```commandline
```

Comment:
Love the example, not blocking here, but we should script handing messages in I reckon.

Reply:
I've [added a script](https://github.com/wellcomecollection/catalogue-pipeline/pull/2735/commits/abf50bf1d3b7972df682808ddd456b3abf322ef3), because I needed to put the [SNS->SQS boilerplate](https://github.com/wellcomecollection/catalogue-pipeline/pull/2735#discussion_r1830850802) in, which is even more ugly and verbose